script/publish_persistent_udfs

```diff
@@ -33,14 +33,14 @@ parser.add_argument(
     "--project-id", default="moz-fx-data-derived-datasets", help="The project ID."
 )
 parser.add_argument(
-    "--dataset",
-    default="udf",
-    help="The name of the dataset the persistent UDFs will be stored in.",
-)
-parser.add_argument(
-    "--udf-dir",
-    default="udf/",
-    help="The directory where declarations of temporary UDFs are stored.",
+    "--udf-dirs",
+    nargs="+",
+    default=["udf/", "udf_js/"],
+    help="""
+    The directories where declarations of UDFs are stored.
+    The names of these directories serve also to specify the target datasets
+    the UDFs get published in.
+    """,
```

Comment:
When help needs to span multiple lines, we've used style like the below elsewhere:

```
        help=(
            "If specified, create views in the target project rather than"
            " the project specified in the file"
        ),
```

That explicitly controls whitespace. It's not clear to me that argparse is going to correctly strip whitespace from a multiline string literal.